### PR TITLE
Allow API tokens to be refreshed, never delete tokens for auditing purposes

### PR DESCRIPTION
**Pull request by nicpottier**

### CODE REVIEW

File: temba/api/migrations/0006_auto_20160617_1610.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    operations = [
+        migrations.AddField(
+            model_name='apitoken',
+            name='is_active',
+            field=models.BooleanField(default=True),
+        ),
+        migrations.AlterUniqueTogether(
+            name='apitoken',
+            unique_together=set([]),
```
Comment (nicpottier) [inline temba/api/migrations/0006_auto_20160617_1610.py]: We used to have a unique on user/role/org, but now that we keep them around and use is_active that isn't valid anymore.

**Comment by norkans7** [comment]
Yep

**Comment by ericnewcomer** [comment]
So you think no way to do it in the UI (save for the api explorer) ?

**Comment by nicpottier** [comment]
screen shot 2016-06-17 at 3 32 28 pm

**Comment by nicpottier** [comment]
Turns blue when you rollover, when you click it a new one is generated is inserted.

**Comment by ericnewcomer** [comment]
oops, see it now. make it so!

**Pull request merged**